topics/fair/tutorials/ro-crate-in-python/tutorial.md

```diff
@@ -105,15 +110,16 @@ table["author"] = alice
 diagram["author"] = bob
 ```
 
-You can also add whole directories together with their contents. In RO-Crate, a directory is represented by the `Dataset` entity:
-
-```bash
-mkdir exp/logs
-touch exp/logs/log1.txt
-touch exp/logs/log2.txt
```

Comment:
I *think* these should also work with `!` prefixes, but, the python version is more pythonic